recipe/meta.yaml

```
@@ -167,12 +193,12 @@ outputs:
         - maven
       host:
         - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
-        - bisonpp  # [not win]
+        - bisonpp  # [linux64]
```

```suggestion
        - bisonpp  # [unix]
```
Same applies below as well.